packages/forms-web-app/src/app.js

```diff
@@ -59,6 +60,30 @@ app
     next();
   });
 
+const isDev = app.get('env') === 'development';
+
+const nunjucksConfig = {
+  autoescape: true,
+  noCache: true,
+  watch: isDev,
+  express: app,
+};
+
+const viewPaths = [
+  path.join(__dirname, '..', 'node_modules', 'govuk-frontend'),
+  path.join(__dirname, '..', 'node_modules', '@ministryofjustice', 'frontend'),
+  path.join(__dirname, 'views'),
+];
+
+const env = nunjucks.configure(viewPaths, nunjucksConfig);
+env.addFilter('appealSiteAddressToArray', appealSiteAddressToArray);
+env.addFilter('date', dateFilter);
+env.addFilter('formatBytes', fileSizeDisplayHelper);
+env.addFilter('filterByKey', filterByKey);
+env.addFilter('addKeyValuePair', addKeyValuePair);
+env.addGlobal('fileSizeLimits', config.fileUpload.pins);
+env.addGlobal('googleAnalyticsId', config.server.googleAnalyticsId);
+
```

Comment:
change looks bigger than it is - had to move this to give access to `env` inside the `initialCookiesMiddleware` below. Surely there is a better way to do this? `res.locals` does not work and cost me several hours trying to make it work.